### DISCUSSION TITLE
Everything's gone

### DISCUSSION THREAD

**Discussion by floupette** [atom/atom#23582] [Q&A]
A moment ago, I opened Atom by accident - I'm clumsy like that - and, when I then closed the application, I was asked to save all of the files, which I didn't because I hadn't edited any of them...
only to realize - a blink too late - Atom had been updated and, apparently, the 1.55 folder had been deleted for good, and my files are now gone. Forever?
Is there any way I can restore them? Thus far it would appear I can't, but I thought I would at least ask.
Anyway, thank you in advance for any - and I mean ANY - help you can provide.

**Comment by icecream17** [comment]
I'm not sure if this could apply here, but are your files in the trash can?

**Reply by floupette** [reply]
Unfortunately, they are not.
